dummy-data_import.py

```diff
@@ -1,112 +1,270 @@
 # import csv data into our demo database
 import csv
```

Comment:
Can this be organized inside a module (directory) to avoid having too much files at the root level structure please.
It could be `/scripts/dummy-data.py`.